### PR TITLE
fixed errors and warnings in Xcode 12

### PR DESCRIPTION
**Pull request by recherst**
fit to Xcode 12 and iOS 14.4

### CODE REVIEW

**Comment by recherst** [comment]
I add this library in my project and can not build successfully. I had solved this problem and some warnings in Xcode 12. please check it as soon as possible cause I'm waiting to use.
thanks!

File: Tests/PhotoLibraryPickerTests/PhotoLibraryPickerTests.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@ final class PhotoLibraryPickerTests: XCTestCase {
         // This is an example of a functional test case.
         // Use XCTAssert and related functions to verify your tests produce the correct
         // results.
-        XCTAssertEqual(PhotoLibraryPicker().text, "Hello, World!")
+//        XCTAssertEqual(PhotoLibraryPicker(<#Binding<[Picture]>#>).text, "Hello, World!")
```
Comment (moifort) [inline Tests/PhotoLibraryPickerTests/PhotoLibraryPickerTests.swift]: You can delete it :)

**Review by moifort — APPROVED** [review]

**Comment by moifort** [comment]
Thanks for your PR. Just one comment :).

File: Tests/PhotoLibraryPickerTests/PhotoLibraryPickerTests.swift (hunk 1 above)
Comment (recherst) [reply Tests/PhotoLibraryPickerTests/PhotoLibraryPickerTests.swift]: > You can delete it :)

all right

**Comment by recherst** [comment]
> Thanks for your PR. Just one comment :).

thanks a lot :)

**Comment by moifort** [comment]
👍🏻👍🏻👍🏻

**Comment by recherst** [comment]
waiting for your new release version.

**Comment by moifort** [comment]
Oh thanks to warn me! I will do it right now!

**Comment by recherst** [comment]
> Oh thanks to warn me! I will do it right now!

Excuse me, is there a new release version? I'm waiting for you...

**Comment by moifort** [comment]
Done!

**Pull request merged**